packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -423,15 +515,127 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     return contentContainer ? contentContainer.offsetHeight - contentContainer.clientHeight : 0;
   }
 
+  private _getScrollbarHeightFromLocalStorage(): number | undefined {
+    if (typeof window !== 'undefined' && 'localStorage' in window) {
+      const scrollbarHeight = Number(window.localStorage.getItem(ScrollablePaneBase._scrollbarHeightKey));
```

Comment:
It feels quite odd to have a component take a direct dependency on local storage. If we want this kind of improvement in calculation, this component should accept as a prop an object which is capable of storing this value. Then we can put a well-tested local storage accessor in utilities.

Reply:
The height/width of scrollbars vary for different browsers, it can't be a prop.

Reply:
@jdhuntington I've removed the localStorage dependency to unblock this PR, please approve.
I'll make the corresponding optimization in future PRs.